scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
Hmmm... I struggled with how to get at the goToSlideNextPrevious & index data without drilling down in the cp data (which wouldn't be fun). I might not be understanding your idea though :/